app/views/datasets/_breadcrumb.html.erb

```diff
@@ -1,26 +1,25 @@
 <% if @dataset %>
-  <div class="grid-row">
-    <!-- Only display breadcrumb if the referrer host matches the application host  -->
-    <div class="column-full">
-      <div class="breadcrumbs">
-        <nav aria-label="Breadcrumb">
-          <ol>
-            <li>
-              <%= link_to t('.home'), root_path, class: 'govuk-link' %>
-            </li>
-            <li>
-              <% if @referer_query.nil? %>
-                <%= @dataset.organisation.title %>
-              <% else %>
-                <%= link_to t('.search'), "/search?#{@referrer}", class: 'govuk-link' %>
-              <% end %>
-            </li>
-            <li aria-current="page">
-              <%= shorten_title(@dataset.title) %>
-            </li>
-          </ol>
-        </nav>
+  <section id="dgu-breadcrumb">
```

Comment:
We tend to use BEM conventions which avoids using ID selectors. Is there a reason why you've switched to or introduced IDs in a few places?